docs/release_process.rst

```diff
@@ -0,0 +1,117 @@
+=====================
+LORE Release Workflow
+=====================
+
+Quick Start
+-----------
+
+prerequisites:
+  - ``git-release-notes`` - https://www.npmjs.com/package/git-release-notes
+  - ``git-flow`` - https://github.com/nvie/gitflow
+
+.. code-block:: bash
+
+    git fetch
+    git checkout master
+    git pull
+    git flow release start 0.10.0
+    vi lore/settings.py # change version number
+    vi util/release_notes_rst.ejs # change version temporarily -- don't commit!
+    # find the hashes for the first and last commits to include
+    git-release-notes v0.9.0..master util/release_notes_rst.ejs
```

Comment:
Does git-release-notes work with tags? I used it with hashes before. For example if first commit is `23b195` and last commit is `7c84a4`

`git-release-notes 23b195..7c84a4 util/release_notes_rst.ejs`